src/ClusterBootstrap/deploy.sh

```diff
@@ -12,7 +12,3 @@ rm -rf deploy/* cloudinit* az_complementary.yaml
 ./cloud_init_deploy.py docker servicesprerequisite
 # execute a deployment action based on the rendered files and machine list got in prerender
 ./cloud_init_aztools.py -v deploy
-# connect the nodes inside the cluster
-./cloud_init_aztools.py interconnect
```

Comment:
yes, merged to add_machines, since I thought we always want to do this after any new deployment. but based on today's scenario, this assumption seems not correct?